centos/common/hpc-tuning.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
 # Disable some unneeded services by default (administrators can re-enable if desired)
-systemctl disable wpa_supplicant
```

Comment:
This will remove these for centos 7.* as well.

Reply:
It is not removed, it is only moved to /azhpc-images/centos/centos-7.x/common/hpc-tuning.sh